java/test/org/openqa/selenium/testing/TestUtilities.java

```diff
@@ -30,7 +30,6 @@
 import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.Platform;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.htmlunit.HtmlUnitDriver;
```

Comment:
Why do we need to remove this?

Reply:
The bazel repin will not remove the asynchttpclient from the maven_install.json when the htmlunit-driver is referenced. I could not fond any use of the htmlunit-driver beside helpers. But yes this is not directly related to this PR.

@titusfortner I will create a new PR for removing the htmlunit-driver tomorrow to see if this is used anywhere.

Reply:
PS: the htmlunit-driver does reference selenium as a dependency and selenium does reference htmlunit-driver as a dependency, what is usually not a good idea

Reply:
I think Selenium should be a dependency and not the other way around. But might make sense to get @rbri input. I'm not sure why things are the way they are now.

Reply:
#12849 it the PR for removing this, the CI on my fork is still passing without.
I think this has been used in the past and has been replaced with real browser tests.